edc-extensions/dataplane/dataplane-token-refresh/token-refresh-core/build.gradle.kts

```diff
@@ -19,6 +19,7 @@
 
 plugins {
     `java-library`
+    `maven-publish`
```

Comment:
is this necessary? I think edc-build plugin is already loading the publish plugin

Reply:
that is correct - removed it.